### PR TITLE
pt-br translation of welcome red hat

### PR DESCRIPTION
**Pull request by caioreis123**
@nodejs/nodejs-pt 
Blog: Announcements. Section 2.2
welcome-redhat.md

### CODE REVIEW

**Comment by Trott** [comment]
@nodejs/nodejs-pt

**Comment by khaosdoctor** [comment]
Looking into it

File: locale/pt-br/blog/announcements/welcome-redhat.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+category: Announcements
+slug: welcome-redhat
+layout: blog-post.hbs
+---
+
+# Fundação Node.js oferece boas vindas à Red Hat como a mais nova membro Platinum
+
+> Empresa procura acelerar a adoção do Node.js para desenvolvimento de software corporativo
+
+**SAN FRANCISCO, 6 de outubro de 2015** - A [Fundação Node.js](https://foundation.nodejs.org/), um consórcio liderado pela comunidade e apoiado pelo mercado, para avançar o desenvolvimento da plataforma Node.js, anunciou hoje que a Red Hat, Inc. ingressou na Fundação como membro Platinum. A Red Hat se une aos membros Platinum, incluindo Famous, IBM, Intel, Joyent, Microsoft e PayPal, para fornecer apoio na adoção, desenvolvimento, e sucesso de longo prazo do projeto Node.js.
```
Comment (JCMais) [inline locale/pt-br/blog/announcements/welcome-redhat.md:14]: Should we translate San Francisco name?

Comment (khaosdoctor) [reply locale/pt-br/blog/announcements/welcome-redhat.md:14]: Nope, San Francisco is a city name

File: locale/pt-br/blog/announcements/welcome-redhat.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+title: Fundação Node.js oferece boas vindas à Red Hat como a mais nova membro Platinum
+date: 2015-10-06T12:30:00.000Z
+status: publish
+category: Announcements
+slug: welcome-redhat
+layout: blog-post.hbs
+---
+
+# Fundação Node.js oferece boas vindas à Red Hat como a mais nova membro Platinum
```
Comment (khaosdoctor) [inline locale/pt-br/blog/announcements/welcome-redhat.md]:
```suggestion
# A Fundação Node.js oferece boas vindas à Red Hat como o mais novo membro Platinum
```

File: locale/pt-br/blog/announcements/welcome-redhat.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+> Empresa procura acelerar a adoção do Node.js para desenvolvimento de software corporativo
```
Comment (khaosdoctor) [inline locale/pt-br/blog/announcements/welcome-redhat.md]:
```suggestion
> A empresa procura acelerar a adoção do Node.js para desenvolvimento de software corporativo
```

**Review by khaosdoctor — CHANGES_REQUESTED** [review]

**Comment by khaosdoctor** [comment]
Just a few changes

**Comment by alexandrtovmach** [comment]
@caioreis123 Could you take a look on requested changes? Thanks

**Comment by caioreis123** [comment]
committed both @khaosdoctor changes and updated the branch, @alexandrtovmach!

**Comment by alexandrtovmach** [comment]
@nodejs/nodejs-pt Please review

File: locale/pt-br/blog/announcements/welcome-redhat.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+Fontes Adicionais
+
+* Saiba mais sobre a [Fundação Node.js](https://foundation.nodejs.org/) e se envolva com [o projeto](https://nodejs.org/en/get-involved/).
+
+## Sobre a Fundação Node.js
+
+A Fundação Node.js. é um projeto colaborativo de código aberto dedicado à criação e suporte à plataforma Node.js. e outros módulos relacionados. O Node.js é usado por dezenas de milhares de organizações em mais de 200 países e acumula mais de 2 milhões de downloads por mês. É o ambiente de execução de escolha para aplicativos de alto desempenho e baixa latência, fornecendo tudo, desde aplicativos corporativos, robôs, mecanismos de API, tecnologias em nuvem e sites para dispositivos móveis. A Fundação é composta por um grupo diversificado de empresas, incluindo membros Platinum Famous, IBM, Intel, Joyent, Microsoft, PayPal e Red Hat. Os membros Gold incluem GoDaddy, NodeSource e Modulus / Progress Software, e os membros Silver incluem Apigee, Codefresh, DigitalOcean, Fidelity, Groupon, nearForm, npm, Sauce Labs, SAP e YLD !. Participe aqui: <https://nodejs.org/>.
```
Comment (jpsc) [inline locale/pt-br/blog/announcements/welcome-redhat.md]:
```suggestion
A Fundação Node.js é um projeto colaborativo de código aberto dedicado à criação e suporte à plataforma Node.js. e outros módulos relacionados. O Node.js é usado por dezenas de milhares de organizações em mais de 200 países e acumula mais de 2 milhões de downloads por mês. É o ambiente de execução de escolha para aplicativos de alto desempenho e baixa latência, fornecendo tudo, desde aplicativos corporativos, robôs, mecanismos de API, tecnologias em nuvem e sites para dispositivos móveis. A Fundação é composta por um grupo diversificado de empresas, incluindo membros Platinum Famous, IBM, Intel, Joyent, Microsoft, PayPal e Red Hat. Os membros Gold incluem GoDaddy, NodeSource e Modulus / Progress Software, e os membros Silver incluem Apigee, Codefresh, DigitalOcean, Fidelity, Groupon, nearForm, npm, Sauce Labs, SAP e YLD !. Participe aqui: <https://nodejs.org/>.
```

Comment (jpsc) [reply locale/pt-br/blog/announcements/welcome-redhat.md]: Removing the . after the first Node.js

**Comment by alexandrtovmach** [comment]
@nodejs/nodejs-pt and, one more review =)

**Comment by Trott** [comment]
> @nodejs/nodejs-pt and, one more review =)

Might want to rebase before reviewing?

File: locale/pt-br/blog/announcements/welcome-redhat.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+O Node.js é o ambiente de execução para aplicativos de alto desempenho e baixa latência, possibilitando tudo, desde aplicativos corporativos a robôs. Nos últimos dois anos, cada vez mais, empresas grandes, incluindo Red Hat, IBM, PayPal, Fidelity e Microsoft, adotaram o Node.js como parte de sua estrutura corporativa. Hoje, existem 2 milhões de endereços IP exclusivos instalando pacotes Node.js. e mais de 2 bilhões de pacotes baixados no último mês.
+
+Freqüentemente usado para criar aplicativos de rede escaláveis e rápidos, o Node.js suporta tecnologias Red Hat, como [Red Hat Mobile Application Platform](https://www.redhat.com/en/technologies/mobile/application-platform), e está disponível em [OpenShift da Red Hat](https://www.openshift.com/) e [Red Hat Software Collections](http://developerblog.redhat.com/tag/software-collections/). Como novo membro, a Red Hat está fornecendo suporte financeiro, contribuições técnicas, e orientações de alto nível para as políticas de diretrizes da recém-criada Fundação que opera como uma organização neutra para apoiar o projeto governado pela comunidade Node.js.
```
Comment (TiagoDanin) [inline locale/pt-br/blog/announcements/welcome-redhat.md:18]:
```suggestion
Frequentemente usado para criar aplicativos de rede escaláveis e rápidos, o Node.js suporta tecnologias Red Hat, como [Red Hat Mobile Application Platform](https://www.redhat.com/en/technologies/mobile/application-platform), e está disponível em [OpenShift da Red Hat](https://www.openshift.com/) e [Red Hat Software Collections](http://developerblog.redhat.com/tag/software-collections/). Como novo membro, a Red Hat está fornecendo suporte financeiro, contribuições técnicas, e orientações de alto nível para as políticas de diretrizes da recém-criada Fundação que opera como uma organização neutra para apoiar o projeto governado pela comunidade Node.js.
```

File: locale/pt-br/blog/announcements/welcome-redhat.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
+---
+title: Fundação Node.js oferece boas vindas à Red Hat como a mais nova membro Platinum
+date: 2015-10-06T12:30:00.000Z
+status: publish
+category: Announcements
+slug: welcome-redhat
+layout: blog-post.hbs
+---
+
+# A Fundação Node.js oferece boas vindas à Red Hat como o mais novo membro Platinum
```
Comment (TiagoDanin) [inline locale/pt-br/blog/announcements/welcome-redhat.md]:
```suggestion
# A Fundação Node.js oferece boas-vindas à Red Hat como o mais novo membro Platinum
```

**Review by TiagoDanin — CHANGES_REQUESTED** [review]

File: locale/pt-br/blog/announcements/welcome-redhat.md (hunk 5 above)
Comment (caioreis123) [reply locale/pt-br/blog/announcements/welcome-redhat.md:18]: i did not  catch the alteration here. could you point the line?

Comment (TiagoDanin) [reply locale/pt-br/blog/announcements/welcome-redhat.md:18]: @caioreis123 `Freqüentemente` replace to `Frequentemente`

**Review by khaosdoctor — APPROVED** [review]
LGTM after the alteration proposed by @tiagodanin

**Comment by ghost** [comment]
ping @nodejs/nodejs-pt, can this be merged now?

**Comment by khaosdoctor** [comment]
@MaledongGit you can merge it then I'll be able to fix it by adding another PR with @TiagoDanin's suggestion. It seems that Caio is less active here now, so I'll make the correction to be faster

**Comment by ghost** [comment]
@khaosdoctor: done ;)

**Pull request merged**